### ISSUE TITLE
keycloak 20.0.3 version Critical vulnerability

### ISSUE THREAD

**Issue by rokkiter** [keycloak/keycloak#16816]
### Description

https://trivy.dev/results/?image=quay.io/keycloak/keycloak:20.0.3
i use trivy scan keycloak lastest version, there are six Critical vulnerability

**Comment by jonkoops** [comment]
@ASzc can we consider this one closed now that #16902 has been merged?

**Comment by abstractj** [comment]
@jonkoops the changes introduced by @ASzc won't solve our issues with CVEs, but instead reduce the number of dependencies.

@rokkiter thanks for submitting your report, but it seems slight inaccurate. Trivy web report does not exclude dependencies under dispute, or without a fix. You can see that most of those dependencies under dispute do not provide a patch, or a fix version. You can see what I mean by running from the command line:

```
$ trivy image --ignore-unfixed quay.io/keycloak/keycloak:nightly
```

For future reports, we would appreciate some previous triage before submitting it. For now, I'm closing it.
